### PR TITLE
Release Notes for 2022-02-01

### PR DESCRIPTION
**Pull request by justindavies**

### CODE REVIEW

**Review by qpetraroia — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+## Release 2022-02-06
+
+This release is rolling out to all regions - estimated time for completed roll out is 2022-02-16 for public cloud and 2022-02-19 for sovereign clouds.
+
+### Announcement
+
+* From Kubernetes 1.23, containerD will be the default container runtime for Windows node pools. Docker support will be deprecated in Kubernetes 1.24. You are advised to test your workloads before Docker deprecation happens by following the documentation here https://docs.microsoft.com/en-us/azure/aks/windows-container-cli#add-a-windows-server-node-pool-with-containerd-preview.
+* Konnectivity rollout will continue in Feb 2022.
+* Kubernetes 1.19 will be removed on the next release.
+* Starting with 1.23 AKS will follow upstream kubernetes and deprecate in-tree azure authentication which is marked for deprecation to be replaced with 'exec'. If you are using Azure CLI or Azure clients, AKS will download kubelogin for users automatically. If outside of Azure CLI, users need to download and install kubelogin in order to continue to use kubectl with AAD authentication. <https://github.com/Azure/kubelogin>
```
Comment (kaarthis) [inline CHANGELOG.md:12]: This is a [client-go credential (exec) plugin](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) implementing azure authentication. This plugin provides features that are not available in kubectl. It is supported on kubectl v1.11+ - Think if we explain the exec plugin with a link it ll be much easier for ppl like me first knowing about this!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+
+### Release Notes
+
+* Behavioral changes
+  * Increase CPU limit on Windows OMS agent from 200mc to 500mc
+* Bug Fixes
+  * Fix azure file NFS mount permissions and enable azure file volume stats by default on AKS 1.21+
+* Preview Features
+  * Kubernetes 1.23.3
+  * Enable ephemeral OS on [temp disk](https://aka.ms/aks/ephemeral-os-temp)
```
Comment (palma21) [inline CHANGELOG.md:23]: Do we have a doc update lined up on the AKS Ephemeral OS docs?
https://docs.microsoft.com/en-us/azure/aks/cluster-configuration#ephemeral-os

We're still calling out the requirement for cache size.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+* Component Updates
+    * Kubernetes 1.20.15, 1.21.9 and 1.22.6 released, 1.20.9, 1.21.2, and 1.22.2 removed
```
Comment (palma21) [inline CHANGELOG.md:25]: 1.19 removed?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,30 @@
...
 
+## Release 2022-02-06
+
+This release is rolling out to all regions - estimated time for completed roll out is 2022-02-16 for public cloud and 2022-02-19 for sovereign clouds.
+
+### Announcement
+
+* From Kubernetes 1.23, containerD will be the default container runtime for Windows node pools. Docker support will be deprecated in Kubernetes 1.24. You are advised to test your workloads before Docker deprecation happens by following the documentation here https://docs.microsoft.com/en-us/azure/aks/windows-container-cli#add-a-windows-server-node-pool-with-containerd-preview.
+* Konnectivity rollout will continue in Feb 2022.
+* Kubernetes 1.19 will be removed on the next release.
```
Comment (palma21) [inline CHANGELOG.md:11]: Can remove?

**Review by palma21 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (justindavies) [reply CHANGELOG.md:11]: Deprecation is still WIP

File: CHANGELOG.md (hunk 3 above)
Comment (justindavies) [reply CHANGELOG.md:25]: WIP

Comment (miwithro) [inline CHANGELOG.md:25]:
```suggestion
    * Kubernetes 1.20.15, 1.21.9 and 1.22.6 released, 1.20.9, 1.21.2, and 1.22.2 removed
    * Upgraded Linux version to 5.4.0-1068.70-azure to address [CVE-2021-4034](https://github.com/Azure/AKS/issues/2756 )
    * Containerd registry configuration for Linux nodes - including adding root CAs for containerd via DS.
```

**Review by miwithro — CHANGES_REQUESTED** [review]

**Review by miwithro — APPROVED** [review]

**Pull request merged**